docs/apache-airflow/ui.rst

```diff
@@ -46,24 +46,39 @@ For example:
 ------------
 
 
-Tree View
+Grid View
 .........
-A tree representation of the DAG that spans across time. If a pipeline is
-late, you can quickly see where the different steps are and identify
+A bar chart and grid representation of the DAG that spans across time.
+The top row is a chart of Dag Runs by duration,
+and below, task instances. If a pipeline is late,
+you can quickly see where the different steps are and identify
 the blocking ones.
 
-There is also visual difference between scheduled and manually triggered
-DAGs/tasks:
+The details panel will update when selecting a DAG Run by clicking on a duration bar:
+
+.. image:: img/grid_run_details.png
+
+Or selecting a Task Instance by clicking on a status box:
+
+.. image:: img/grid_instance_details.png
+
+Manual runs are indicated by a play icon (just like the Trigger DAG button):
 
 .. image:: img/task_manual_vs_scheduled.png
 
-The DAGs/tasks with a black border are scheduled runs, whereas the non-bordered
-DAGs/tasks are manually triggered, i.e. by ``airflow dags trigger``.
+Task groups are indicated by a caret and can be opened or closed
+
+.. image:: img/grid_task_group.png
+
+Mapped Tasks are indicated by a square brackets and will show a table of each mapped instance in the details panel:
+
+.. image:: img/grid_mapped_task.png
+
 
 
 ------------
 
-.. image:: img/tree.png
+.. image:: img/grid.png
```

Comment:
I wonder if we should move this up to be the first screenshot.

Reply:
Isn't the first screenshot the DAGs view of the homepage? I feel like we should leave that as-is.

Reply:
Sorry, more specifically the first screenshot in the grid section.